qemu/tests/qemu_mem_utilization.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import time
+
+from virttest import utils_test
+from virttest import error_context
+from autotest.client.shared import utils
+from autotest.client.shared import error
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Compare host used memory after guest boot/reboot/shutdown for times:
+    1) boot guest
+    2) after guest up, check qemu memory utilization
+    3) reboot guest for times
+    4) check qemu memory utilization
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def mem_utilization(timeout):
+        cmd = params.get("host_mem")
```

Comment:
No.
params['host_mem']: if 'host_mem' is empty, there will be error directly. it's always used for those must params.
Instead, there will no error for params.get('host_mem') although 'host_mem' is empty or not existed.